main.cpp

```diff
@@ -1,5 +1,6 @@
 #include <iostream>
-
+#include <cmath>
+using namespace std;

```

Comment:
Never use `using namespace std;`
it just causes problems when your project gets big.
- [ ] revise

Reply:
Done! what happens if I leave it? I actually just added it because I saw it was needed to use wchar_l but I couldn't get rid of the warning message.

Reply:
Don't include code that isn't used.  it just adds bloat and confusion later on when you revisit the project.